helm/charts/determined/scripts/k8s-password-change.py

```diff
@@ -36,54 +31,72 @@ def pwChange(user: str, password: str, entrypoint: str) -> None:
 def checkPortAlive(entrypoint: str) -> None:
     while True:
         try:
-            api.get(entrypoint, "/api/v1/master", authenticated=False)
+            requests.get(
+                url=f"http://{entrypoint}/api/v1/master",
+                verify=False,
+            )
             return
         except Exception as e:
             print(f"Encountered exception: {e}")
             continue
 
 
-def getMasterAddress(namespace: str, service_name: str, master_port: str, node_port: str) -> str:
-    config.load_incluster_config()
-    v1 = client.CoreV1Api()
+def getMasterAddress(
+    namespace: str,
+    service_name: str,
+    master_port: str,
+    node_port: str,
+    service_host: str,
+    https_host: str,
```

Comment:
since this comes from `KUBERNETES_PORT_...`, can we rename it to

```suggestion
    service_port: str,
```

Reply:
fixed!